Screen-sLaYeR.md

```diff
@@ -0,0 +1,52 @@
+---
+name: "@Screen-sLaYeR"
+project: "Robotic Arm"
+---
+
+# Robotic Arm
+
+## Summary
+
+what are ou going to build?
+A three axis Robotic Arm. 
+
+What does it do?
+It pick up things and does many other stuff. 
+
+Why are you excited to build it?
+It's my third Electronic Project and it can help me on my Desktop
+
+## Plan
+What steps are you going to take to build it?
+1. Making the Circuit Diagram
+2. Getting the Parts
+3. Connecting the parts
+4. The Programming Part
+
+What tools are you going to use?
+1. Soldering Iron (My Elder Sister own that)
+2. 3D Printer for Printing the Parts or Cutting it out from PVC (Cutting is Preffered as I don't have 3D Printer)
+
+What will you do first, second, third, etc.?
+1. Get The Parts 
+2. Assemble the Servos and Connect all of them Properly
+3. Proogram it to Work Properly
+
+[Robotic Arm STL Files](https://cloud-qxhtc1x05-hack-club-bot.vercel.app/0arduino_robot_arm_stl_files.zip)
+
+## Budget
+
+| Product                   | Supplier/Link                                                                             | Cost   |
+| ---------------           | -------------------------------------                                                     | ------ |
+| Pack of wires             |https://www.amazon.in/gp/product/B071VQLQQQ/ref=ox_sc_act_image_8?smid=AH017Z3M1ZJ3T&psc=1 | $1.95  |
+| 5V  - 2A Adapter          |https://www.amazon.in/gp/product/B09BZXFHNL/ref=ox_sc_act_image_3?smid=A32I52VHFPY5IG&psc=1| $2.40  |
+| Stepper Motor x 2         |https://www.amazon.in/gp/product/B08J8C3TJS/ref=ox_sc_act_image_1?smid=ARY8SMPB4PT5P&psc=1 | $16.88 |
```

Comment:
what's the role of the stepper motors?

Reply:
The 1st will be added at the base for a complete circular rotation and the other at the claw

Reply:
ooo cool! and is this instead of a servo? thank you sm for taking the time to help me understand this :)

also, could you please add what you're modifying within this file?

Reply:
> ooo cool! and is this instead of a servo? thank you sm for taking the time to help me understand this :)
> 
> also, could you please add what you're modifying within this file?

Yes that is Instead of Servo, and also so Sorry I have messed with the Quantities of Servos, I will Fix that 
+
Sure, I will add what I planned to Modify

Reply:
no worries! thank you sm :)